### ISSUE TITLE
Validator::make() function does not validate 'unique' rule while model update scenario

### ISSUE THREAD

**Issue by naveedali8086** [laravel/framework#32066]
- Laravel Version: 6.5.2
- PHP Version: 7.2.19
- MYSQL 5.7.24

### Description:
I am using 'unique' rule with Validator::make() function. (I can use $this->validate() in controller but due to some reason i am having to use Validator::make() function, anyway)

Validator::make() does not works according to the expections while editing a model while it works perfect while creating a model.

**Model Create Case: (Works fine)**
Validator::make($request->all(), ['number' => "unique:racks"]

**Model Update Case: (Does not works fine)**
Validator::make($request->all(), ['number' => "unique:racks,id,$id"]

When I pass column name and the value of column as above, Laravel does not validate the request and let it go further into the app and raises Database exception (SQLSTATE[23000]: Integrity constraint violation)

When I try the same rule 'unique:racks,id,$id' in update scenario with $this->validate() method in controller, Laravel validation WORKS fine and when I use 

$this->validate($request, ['number' => "unique:racks"]) in update scenario (note that I have removed the column name and its value from rule) laravel shows errors. i.e Laravel is checking uniqueness on the same model

**In short:**

**Model Create Case:** 
- Validator::make($request->all(), ['number' => "unique:racks"] ==> works fine
- $this->validate($request, ['number' => "unique:racks"])        ==> works fine

**Model Update Case:**
- Validator::make($request->all(), ['number' => "unique:racks,id,$id"]  ==> **Does not** works fine
- $this->validate($request, ['number' => "unique:racks,id,$id"])        ==> works fine (and note If I remove column name and its value from rule, then it does not work as expection while Validator::make() works)

**Comment by staudenmeir** [comment]
Your second validation rule is incorrect, the column name is "missing" (i.e. `id` is used instead):

```php
Validator::make($request->all(), ['number' => "unique:racks,number,id,$id"]
```

**Comment by naveedali8086** [comment]
@staudenmeir , There is a typo in the error description described above but the error is still there.

I have retested the model edit case couple of times but the problem is same. i.e. instead of generating proper validation error, laravel is throwing exception generated by the database.

In edit case, the following line skips unique check on the current model (that is correct) but I we intentionally give some value that already exist in the DB, then the following line produces error. 

`Validator::make($request->all(), ['number' => "unique:racks,id,$id"]`

**For example:**
user 1 has example**1**@example.com while
user 2 has example**2**@example.com

Now If we open the **user2** in edit form and update its email with **user1's email**, then the following rule throws exception instead of generating the proper validation error.

`Validator::make($request->all(), ['email' => "unique:users,id,$id"]`

I hope this will clarify you.

@themsaid , I have retested the edit model case couple of times, but the error is still there, please have a look at it.

Thanks

**Comment by staudenmeir** [comment]
> Validator::make($request->all(), ['number' => "unique:racks,id,$id"]

This is still the incorrect rule, not the one I posted. The same goes for the second rule.

**Comment by naveedali8086** [comment]
@staudenmeir , Can you please let me know what's wrong with above rule.

p.s. $id will be replace by actual numeric value of the model and you would know that a variable can be passed in double quotations.

Thank you
